services/models/route_rule.go

```diff
@@ -51,4 +53,36 @@ func (rule *RouteRule) Parse(routeRule kubernetes.IstioObject) {
 	rule.Mirror = routeRule.GetSpec()["mirror"]
 	rule.CorsPolicy = routeRule.GetSpec()["corsPolicy"]
 	rule.AppendHeaders = routeRule.GetSpec()["appendHeaders"]
+
+	rule.detectRouteMisconfiguration()
+}
+
+func (rule *RouteRule) detectRouteMisconfiguration() {
```

Comment:
Not sure if I understand this one. Isn't the logic to detect a mis-configuration delegated into a checker object ?
This sounds like logic is defined twice ?

Or perhaps I missed something.

Reply:
Oopsie... good seen! This is code that shouldn't be here anymore. This was an old proposal. Thanks for catch it!